src/globus_sdk/_remarshal.py

```diff
@@ -0,0 +1,159 @@
+"""
+This module provides internal helpers for remarshalling data from one shape
+into another.
+
+This may be as simple as converting strings from one form to another, or as
+sophisticated as building a specific internal object in a configurable way.
+"""
+
+from __future__ import annotations
+
+import collections.abc
+import typing as t
+import uuid
+
+from globus_sdk._missing import MISSING, MissingType
+from globus_sdk._types import UUIDLike
+
+T = t.TypeVar("T")
+R = t.TypeVar("R")
+
+
+def safe_strseq_iter(
```

Comment:
I don't think this method should be named `safe`.

All other functions in the module use the prefix `safe` to convey "if the value you pass is Missing or None, we'll just return that" except for this one.

Not sure if there's some history around this naming discrepency, but it leads to developer confusion when a term is almost-perfectly consistent.

Reply:
I've done some significant renaming in my latest commit, introducing `Omittable[T]` and `NullableOmittable[T]` to describe these types.

Let me know if you want more changes or if we can consider this sufficiently refined.